HW_1/task_2.py

```diff
@@ -0,0 +1,17 @@
+# 2-я задача из домашнего задания №1
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. 
+# Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, 
+# если известно, что Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# Пример:
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+s = int(input('Введите общее кол-во журавликов: '))
+a = s / 6
+b = s * 2 / 3
+c = s / 6
+if (s % 6 == 0 ):
+    print(int(a), int(b), int(c))
```

Comment:
На мой взгляд, сначала нужно было бы проверить входные данные на кратность 6, а уже потом вычислять переменные.
```suggestion
if s % 6 == 0 :
    a = с == s // 6
    b = s * 2 // 3
    print(a, b, c)
```